jupyterdrive/gdrive/gapi_utils.js

```diff
@@ -208,21 +223,38 @@ define(function(require) {
                     error.name = 'GapiError';
                     return Promise.reject(error);
                 }
-                return authorize(true);
+                return authorize(true, config);
             });
         }
     };
 
+    var _handle = {};
+    var _conf_prm = new Promise(function(resolve){
+        _handle.resolve = resolve;
+    })
+
+
+    var config = function(conf){
```

Comment:
Can you add a comment like "calling config with conf, results in the promise _conf_prm being resolved with conf.  This then triggers the rest of the gapi loading".